[debug] [2019-07-17T16:51:10.577Z] ----------------------------------------------------------------------
[info] 
[info] Visit this URL on any device to log in:
[info] https://accounts.google.com/o/oauth2/auth?client_id=563584335869-fgrhgmd47bqnekij5i8b5pr03ho849e6.apps.googleusercontent.com&scope=email%20openid%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloudplatformprojects.readonly%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Ffirebase%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloud-platform&response_type=code&state=930464603&redirect_uri=http%3A%2F%2Flocalhost%3A9005
[info] 
[info] Waiting for authentication...
[debug] [2019-07-17T16:51:24.274Z] >>> HTTP REQUEST POST https://accounts.google.com/o/oauth2/token  

[debug] [2019-07-17T16:51:24.414Z] Token Fetch Error: Error
    at new FirebaseError (C:\Users\bhatns1\AppData\Roaming\npm\node_modules\firebase-tools\lib\error.js:9:18)
    at Request._callback (C:\Users\bhatns1\AppData\Roaming\npm\node_modules\firebase-tools\lib\api.js:30:31)
    at self.callback (C:\Users\bhatns1\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\request\request.js:185:22)
    at Request.emit (events.js:198:13)
    at Request.EventEmitter.emit (domain.js:448:20)
    at Request.onRequestError (C:\Users\bhatns1\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\request\request.js:881:8)
    at ClientRequest.emit (events.js:198:13)
    at ClientRequest.EventEmitter.emit (domain.js:448:20)
    at TLSSocket.socketErrorListener (_http_client.js:392:9)
    at TLSSocket.emit (events.js:198:13)
